lib/partitioninfo.coffee

```diff
@@ -1,42 +1,57 @@
 _ = require('lodash')
 filedisk = require('file-disk')
 MBR = require('mbr')
+GPT = require('gpt')
 Promise = require('bluebird')
 
 ###*
 # @module partitioninfo
 ###
 
-BOOT_RECORD_SIZE = 512
+MBR_SIZE = 512
+GPT_SIZE = 512 * 33
```

Comment:
Is there a way we can get these constants from the `gpt` or `mbr` module? If they are not there, can we add them? @jhermsmeier ?

Reply:
It looks like mbr and gpt do not export anything like this.